src/avt/Expressions/CMakeLists.txt

```diff
@@ -362,7 +365,6 @@ ${VISIT_SOURCE_DIR}/visit_vtk/lightweight
 ${VISIT_SOURCE_DIR}/vtkm
 ${VISIT_SOURCE_DIR}/third_party_builtin/verdict
 ${VTK_INCLUDE_DIRS}
-${VTKM_INCLUDE_DIR}
```

Comment:
Your modification comment for this block says: "I replaced vtkh with vtkm."
But it looks like all you did was remove a vtkm include.

It doesn't matter... but I noticed it 👀